src/Arcus.Scripting.LogicApps/Scripts/Disable-AzLogicAppsFromConfig.ps1

```diff
@@ -53,59 +57,137 @@ function ExecuteCheckType() {
         }
         if ($batch.checkType -Match "NoWaitingOrRunningRuns") {
             Write-Host "Executing Check 'NoWaitingOrRunningRuns'"
-            if ($batch.logicApps.Length -gt 0 ) {
-                $batch.logicApps | ForEach-Object {
-                    $logicApp = $_;
-                    if($ResourcePrefix.Length -gt 0){
-                        $logicApp = "$ResourcePrefix$_"
-                    }
+            if ($batch.logicAppType -match "Standard") {
+                if ($batch.logicApps.Length -gt 0 ) {
+                    $batch.logicApps | ForEach-Object {
+                        $LogicAppName = $_.name;
 
-                    try {
-                        $runHistory = Get-AzLogicAppRunHistory -ResourceGroupName $ResourceGroupName -Name $logicApp -FollowNextPageLink -MaximumFollowNextPageLink $maximumFollowNextPageLink -ErrorAction Stop
-                        $RunningRunsCount = ($runHistory | Where-Object { $_.Status -eq "Running" }).Count
-                        $WaitingRunsCount = ($runHistory | Where-Object { $_.Status -eq "Waiting" }).Count
-                        if ($RunningRunsCount -ne 0 -and $WaitingRunsCount -ne 0) {
-                            while ($RunningRunsCount -ne 0 -and $WaitingRunsCount -ne 0) {
-                                Write-Verbose "Azure Logic App '$logicApp' has Running and/or Waiting Runs, waiting 10 seconds and checking again..."
-                                Write-Debug "Number of running runs: $RunningRunsCount"
-                                Write-Debug "Number of waiting runs: $WaitingRunsCount"
-                                Start-Sleep -Second 10
-                                $runHistory = Get-AzLogicAppRunHistory -ResourceGroupName $ResourceGroupName -Name $logicApp -FollowNextPageLink -MaximumFollowNextPageLink $maximumFollowNextPageLink -ErrorAction Stop
-                                $RunningRunsCount = ($runHistory | Where-Object { $_.Status -eq "Running" }).Count
-                                $WaitingRunsCount = ($runHistory | Where-Object { $_.Status -eq "Waiting" }).Count
-                                if ($RunningRunsCount -eq 0 -and $WaitingRunsCount -eq 0) {
-                                    Write-Verbose "Found no more waiting or running runs for Azure Logic App '$logicApp', executing stopType for Logic App"
-                                    ExecuteStopType -resourceGroupName $ResourceGroupName -LogicAppName $logicApp -stopType $batch.stopType
+                        if ($_.workflows.Length -gt 0 ) {
+                            $_.workflows | ForEach-Object {
+                                $WorkflowName = $_;
+
+                                $listRunningUrl = . $PSScriptRoot\Get-AzLogicAppStandardResourceManagementUrl.ps1 -EnvironmentName $EnvironmentName -SubscriptionId $Global:subscriptionId -ResourceGroupName $ResourceGroupName -LogicAppName $LogicAppName -WorkflowName $WorkflowName -Action 'listRunning'
```

Comment:
Think this script still uses the `-Global...` stuff?

Reply:
As discussed, this script already used the global settings, for backwards compatibility reasons we will keep this in for now and in the future remove it 👍🏻